CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Changelog for NeoFS Node
 
 ### Updated
 - SDK to the post-api-go version (#3103)
+- neofs-contracts to 0.21.0 (#3157)
```

Comment:
neo-go worth mentioning?

Reply:
Yes! In fact, there is an important thing to keep in mind for this upgrade, let me add it.